app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    cached_results_storage = {}
+
+    def inner(*args, **kvargs) -> int:
+        if args in cached_results_storage:
+            print("Getting from cache")
+            return cached_results_storage.get(args)
```

Comment:
U don't need to use `get` here. You can be sure that there will be such key, you did this check in if statement